staging/src/k8s.io/component-base/cli/globalflag/globalflags.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/spf13/pflag"
 
 	"k8s.io/component-base/logs"
+	// ensure libs have a chance to globally register their flags
+	_ "k8s.io/klog"
```

Comment:
which libs?

Reply:
see https://github.com/kubernetes/kubernetes/pull/71491/commits/3ecf79ea11b57788ae5ac0b7fe4bbc9d2810d346#r269949965. But, i think no need any more now, for we discard `global := flag.CommandLine`

Reply:
I would expect components to import the libraries that register global flags in the same package they use globalflag from. We can't possibly centralize everyone's dependencies here.